pkgs/shells/zsh/zsh-fzf-history-search/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, substituteAll, pkgs, bash, fzf }:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```
I think `stdenvNoCC` is better here because this derivation only copies the shell scripts.